components/browser/toolbar/src/main/java/mozilla/components/browser/toolbar/display/DisplayToolbar.kt

```diff
@@ -130,7 +130,7 @@ internal class DisplayToolbar(
         }
     }
 
-    private val progressView = ProgressBar(
+    private val progressView = AnimatedProgressBar(
```

Comment:
I strongly suggest to not do that. `AnimatedProgressBar` utillizes the CPU a lot (for the animation). This negatively affects page load performance since the CPU is busy with animating.

Also note that apps can use their own progress bar if they wish to. Fire TV does that and we even have a sample with a custom progress bar:
https://github.com/mozilla-mobile/android-components/blob/master/samples/toolbar/src/main/java/org/mozilla/samples/toolbar/ToolbarActivity.kt#L294